content/S2015/S15-event-alterconf.md

```diff
@@ -0,0 +1,26 @@
+iTitle: WiCS is going to AlterConf Toronto! 
+Date: 2015-07-11 9:00
+Category: Events
+Tags: social
+Slug: alterconf-toronto-2015
+Author: Fatema Boxwala
+Summary: Hitch a ride with WiCS to AlterConf in Toronto!
+
+WiCS has chartered a bus to transport Univeristy of Waterloo students to
+AlterConf on Saturday July 11 2015! Space is limited so be sure to
+[register](https://goo.gl/86ILSn) and RSVP to the FaceBook event below.
```

Comment:
There's no link to the Facebook event. Both links are to the registration form (and one to the alterconf site).